spec/inputs/elasticsearch_spec.rb

```diff
@@ -52,6 +52,19 @@
       end
     end
 
+    describe 'handling obsolete settings' do
```

Comment:
Inspired by tests in https://github.com/logstash-plugins/logstash-input-http/pull/182 credit @robbavey